caffe2/proto/torch.proto

```diff
@@ -416,11 +466,36 @@ message TypeProto {
     optional TensorShapeProto shape = 2;
   }
 
+  // Sequence type: List, Tuple
+  message Sequence {
+    // Type for each element.
```

Comment:
I think we need two different cases for list and tuple; all elements in a list are of the same type, but in a tuple, we each element can be of a different type.

Reply:
I see. Let me modify this part.